hack/update-generated-swagger-spec.sh

```diff
@@ -61,8 +61,17 @@ popd > /dev/null
 wait_for_url "${HOST}/healthz" "apiserver: " 0.25 80
 
 echo "Updating ${SWAGGER_SPEC_OUT_DIR}:"
-echo "Updating ${SWAGGER_SPEC_OUT_DIR}/oapi-v1.json from ${SWAGGER_API_PATH}oapi/v1..."
-curl "${SWAGGER_API_PATH}oapi/v1" > "${SWAGGER_SPEC_OUT_DIR}/oapi-v1.json"
-echo "Updating ${SWAGGER_SPEC_OUT_DIR}/api-v1.json from ${SWAGGER_API_PATH}api/v1..."
-curl "${SWAGGER_API_PATH}api/v1" > "${SWAGGER_SPEC_OUT_DIR}/api-v1.json"
+OAPI_ENDPOINTS=$(curl "${HOST}" | grep -Po "(?<=\/oapi\/)[a-z0-9]+" | sed '/v1beta3/d')
```

Comment:
seems like api and oapi should be passed to a loop?  These are the same other than that, right?